.github/workflows/ci.yaml

```diff
@@ -40,7 +41,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
-          python-version: 3.10
+          python-version: 3.13
```

Comment:
😅 phew, anything but 3.10 unquoted, as yaml passes that to setup-python as 3.1 (float) 😇 so this one's safe now;)

Reply:
Ah, so that should be quoted then? I agree everywhere else we should have the base python used. But here it's doing the build so whatever current makes sense to me.

Reply:
Not necessarily, unless needed to disambiguate (e.g. 4.0 float gets often passed as int 4 which could mean 4.x for many tools… yea version "numbers" are actually not numbers, but mostly need treating as strings… so as long as you're aware of that you're fine.) ~ I usually quote that whenever I see 3.9 used, to make sure the next person to bump it won't make the runner implode;D… but once you're past that it's not that big of a foot gun.

But folks were indeed [surprised](https://github.com/actions/setup-python/issues/160#issuecomment-724485470) back then;)